aem-authoring-toolkit-assets/src/main/content/jcr_root/apps/authoring-toolkit/depends-on/README.md

```diff
@@ -42,6 +42,7 @@ For referenced field:
 
 * `data-dependsonref` - to mark a field, that is referenced from the Query.
 * `data-dependsonreftype` - (optional) to define expected type of reference value. 
+* `data-dependsonreflazy` - (marker) attribute to mark reference as lazy. DependsOn doesn't observe rapid events like `input`.
```

Comment:
(cosmetic) maybe rephrase the lase sentence as "In this case, DependsOn will not observe rapid events like `input`."